src/Lykke.Service.Dash.Job/PeriodicalHandlers/BalanceHandler.cs

```diff
@@ -11,8 +11,8 @@ public class BalanceHandler : TimerPeriod
         private ILog _log;
         private IPeriodicalService _periodicalService;
 
-        public BalanceHandler(int period, ILog log, IPeriodicalService periodicalService) :
-            base(nameof(BalanceHandler), period, log)
+        public BalanceHandler(TimeSpan period, ILog log, IPeriodicalService periodicalService) :
+            base(nameof(BalanceHandler), (int)period.TotalMilliseconds, log)
```

Comment:
Optional. You could use ```TimerTrigger``` class via aggregation instead of ```TimerPeriod```. ```TimerTrigger``` accepts ```TimeSpan``` as period, not ```int```

Reply:
I see. Will use it next time. Thanks for the info